src/Configuration/Configuration.php

```diff
@@ -94,12 +92,13 @@ private function __construct(array $options)
             throw InvalidConfigurationException::fromUnknownLayerNames(...$unknownLayerNames);
         }
 
+        $options['ruleset']['skip_violations'] = $options['skip_violations'];
```

Comment:
This is not a good idea, because `ruleset.skip_violations` could be a legitimate ruleset. Can we instead make it second and third parameters for `ConfigurationRuleset::fromArray`?

Reply:
@patrickkusebauch 
Sure thing.
But does it not mean it shouldn't be called `ConfigurationRuleset::fromArray` anymore?

Reply:
Whatever name floats your boat. :smile: I would probably use something like `fromOptions`

Reply:
@patrickkusebauch 
Refactored to passing the fields as parameters to  `ConfigurationRuleset::fromOptions`